pioinstaller/python.py

```
@@ -153,104 +151,133 @@ def check():
     if any(s in util.get_pythonexe_path().lower() for s in ("msys", "mingw", "emacs")):
         raise exception.IncompatiblePythonError(
             "Unsupported environments: msys, mingw, emacs >> %s"
             % util.get_pythonexe_path(),
         )
 
     try:
-        assert os.path.isdir(os.path.join(sys.prefix, "Scripts")) or (
-            sys.version_info >= (3, 5) and __import__("venv")
-        )
-    except (AssertionError, ImportError):
+        assert os.path.isdir(os.path.join(sys.prefix, "Scripts"))
```

Ruff (0.12.2)

Use of `assert` detected

(S101)